agent/internal/agent.go

```diff
@@ -221,8 +221,10 @@ func (a *agent) setup(ctx *actor.Context) error {
 		}
 	}
 
-	if err := a.getMasterInfo(); err != nil {
-		return errors.Wrap(err, "error fetching master info")
+	if a.MasterHost != "" {
```

Comment:
Lets leave this change out of this PR.

Reply:
yeah sure, sorry, I just needed it for how I was testing and forgot to remove.